charts/fluid/fluid/templates/controller/alluxioruntime_controller.yaml

```diff
@@ -43,7 +43,7 @@ spec:
             value: {{ .Values.workdir | quote }}
           {{- end }}
           {{- if .Values.runtime.alluxio.init.image }}
-          - name: ALLUXIO_INIT_IMAGE_ENV
+          - name: DEFAULT_INIT_IMAGE_ENV
             value: {{ .Values.runtime.alluxio.init.image | quote }}
```

Comment:
Since the init image is no longer an Alluxio-specific one, how about moving the helm chart value outside the `runtime.alluxio` scope? I am wondering if there's some compatibility issue. What do you think? @yangyuliufeng

Reply:
I think we don't need to change it.
Even if  the init image is no longer an Alluxio-specific one, different runtimes can use or not use it.
So, runtime.xxx.init is still needed.